examples/auto_deploy/model_registry/configs/deepseek_v3_lite.yaml

```
@@ -0,0 +1,3 @@
+# Configuration for DeepSeek V3 and R1 with reduced layers
+# Full models are too large, so we test with limited layers
+model_kwargs: .num_hidden_layers=10
```

```suggestion
model_kwargs:
  num_hidden_layers: 10
```

Let's use the proper yaml notation